### DISCUSSION TITLE
Enhancement request: cron.present - change default identifier to UUID

### DISCUSSION THREAD

**Discussion by udf2457** [saltstack/salt#67659] [Feature Requests]
At the moment the `identifier` variable in `cron.present` defaults to `name`.

`name` being _"The command that should be executed by the cron job."_

Unfortunatley the logic in SaltStack does not appear to be intelligent enough to deal with changes in the command to be executed in cron.

This results in a default behaviour of a second entry being made in cron for the "new" job, and the "old" job being left in place. This is very much undesireable default behaviour for obvious reasons !

It seems to me that a sensible solution to the problem is to default `identifier` to a UUID instead of `name`.

**Comment by sagetherage** [comment]
SaltStack is the company the product is Salt so of course the logic of the company does very little, here. A common misconception on the name, tagging for review of future feature work.
